rsmtool/modeler.py

```diff
@@ -34,14 +35,37 @@ class Modeler:
     """Class to train model and generate predictions with built-in or SKLL models."""
 
     def __init__(self, logger=None):
-        """Instantiate empty instance with no learner and given logger, if any."""
+        """
+        Initialize empty ``Modeler`` object.
+
+        The learner and the scaling-/trimming-related attributes are
+        set to ``None``.
+        """
         self.learner = None
+        self.trim_min = None
+        self.trim_max = None
+        self.trim_tolerance = None
+        self.train_predictions_mean = None
+        self.train_predictions_sd = None
+        self.h1_mean = None
+        self.h1_sd = None
         self.logger = logger if logger else logging.getLogger(__name__)
 
+    def save(self, model_path):
+        """
+        Save an instance of this class to disk.
+
+        Parameters
+        ----------
+        model_path : str
+            Destination path for model file
+        """
+        joblib.dump(self, model_path)
+
     @classmethod
     def load_from_file(cls, model_path):
         """
-        Load a modeler object from a file on disk.
+        Load a ``Modeler`` object from a file on disk.
 
         Parameters
         ----------
```

Comment:
Shouldn't the mention of modeler in model_path description get the backticks and capitalization also?

Reply:
Also the same thing under returns:  A ``Modeler`` instance.

Reply:
I have gone through a lot of the documentation and tried to make things more consistent. It's not always the case, though, that I'm referring to `Modeler` or a `modeler` (that is, if that's the name of the parameter itself, in which case I'm not talking about the class). I think it should be relatively clear now.